RecoHI/HiJetAlgos/plugins/HiBadParticleCleaner.cc

```diff
@@ -91,7 +92,11 @@ void HiBadParticleCleaner::produce(edm::StreamID, edm::Event& iEvent, const edm:
 
   bool foundBadCandidate = false;
 
+  int n = pfCandidates->size();
+  std::vector<int> oldToNew(n);
```

Comment:
It's temporary in the sense that it's only needed for reMiniAOD of the existing 2018 PbPb AOD.  If we were to re-reconstruct the 2018 data, we wouldn't need this.  We will also not need it for Run 3 (unless we screw up the track quality selections again).

Reply:
If this has to be a definitive fix I am convinced that a more meaningfull name than `oldToNew` could be invented.
Since the size is known, it could also probably more conveniently became a` std::array<int, n>`